packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -170,6 +176,37 @@ describe('logs', () => {
       expect(server.requests.length).toEqual(0)
       expect(sendSpy).toHaveBeenCalled()
     })
+
+    it('should log reports if ff "forward-reports" enabled', () => {
+      if (!isChromium()) {
+        pending('no ReportingObserver support')
+      }
+      const errorLogger = new Logger(noop)
+      const logErrorSpy = spyOn(errorLogger, 'error')
+      const reportingObserverStub = stubReportingObserver()
+      updateExperimentalFeatures(['forward-reports'])
+      startLogs({ errorLogger })
+
+      reportingObserverStub.raiseReport()
+
+      expect(logErrorSpy).toHaveBeenCalled()
+
+      resetExperimentalFeatures()
+      reportingObserverStub.reset()
+    })
+
+    it('should not log reports if ff "forward-reports" disabled', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should not this test be skipped on non chromium browsers?

Reply:
sure